frontend/packages/dev-console/src/components/topology/D3ForceDirectedRenderer.tsx

```diff
@@ -704,11 +704,17 @@ export default class D3ForceDirectedRenderer extends React.Component<
   };
 
   private initializeMoveConnection = (edge: ViewEdge) => {
-    const { edgesById } = this.state;
+    const { edgesById, nodes } = this.state;
 
     const outwardEdges = _.filter(edgesById, (viewEdge: ViewEdge) => {
       return viewEdge.source.id === edge.source.id;
     });
+
+    // Only initiate moving the connection if there are other nodes available to connect to
+    if (nodes.length > outwardEdges.length + 1) {
```

Comment:
Sorry, my prior logic was flawed and this doesn't allow dragging of connectors unless you ought not.

Should be:
```suggestion
    if (nodes.length <= outwardEdges.length + 1) {
```

Also, need to check for the moveConnection existance in the `onMoveConnectionDragged` and `onMoveConnectionEnd` calls and bail if not really dragging (due to return above).

Reply:
Done